cetmix_tower_yaml/models/cx_tower_yaml_mixin.py

```diff
@@ -110,8 +120,59 @@ def _post_process_record_values(self, values):
             values.update(
                 {"access_level": self.TO_YAML_ACCESS_LEVEL[values["access_level"]]}
             )
+        # Post process m2o fields
+        for key, value in values.items():
+            if key.endswith("_id"):
+                processed_value = self._process_m2o_value(key, value, record_mode=True)
+                values.update({key: processed_value})
```

Comment:
what if processed_value returned None, because the model is not support yaml? from this eval `if comodel_name and self._model_supports_yaml(comodel_name):`

Reply:
Then None value will be assigned to the key. Which is fine

Reply:
@ivs-cetmix in this case I suggest to return False explicitly so that there is no confusion in reading the code